System.Containers/Configuration.cs

```diff
@@ -1,10 +1,56 @@
-﻿namespace System.Containers;
+﻿using System.Diagnostics;
 
-public static class Configuration
+namespace System.Containers;
+
+public static class ContentStore
```

Comment:
We should rename the file too

Reply:
Gah, VS was supposed to do that automagically